.github/workflows/containers.yaml

```diff
@@ -77,23 +75,27 @@ jobs:
           ${{ matrix.dockerfile_suffix }}:${tag} \
             --help
 
+      # ${{ github.repository_owner }}
+      #   It goes to that new namespace (aka destination) - upstream
+      #   Pull Request (github.event_name: pull_request)
+      # ${{ github.actor }}
+      #   The person who open'd up it up (aka source)
+      #   Push Request (github.event_name: push)
       - name: Login to GitHub Packages (ghcr.io)
         if: |
-          ${{ github.event_name != 'pull_request' }} &&
-          ${{ github.ref == 'refs/heads/main' }}
+          github.event_name != 'pull_request'
         uses: redhat-actions/podman-login@v1
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: >
-          Push ${{ env.container }} image to
```

Comment:
This seems repetitive - the original comment is clearer

Reply:
Dropped